cibuild.sh

```diff
@@ -299,6 +299,12 @@ if [[ $BOOTSTRAP_ONLY = true ]]; then
     exit $?
 fi
 
+# Microsoft.Net.Compilers package is available now, so we can use the latest csc.exe
+if [ "$host" = "Mono" ]; then
+        CSC_EXE=`find $PACKAGES_DIR/microsoft.net.compilers -name csc.exe`
```

Comment:
I hate to agree but I think I do--when we change the compiler version there are likely to be two side by side here.